src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age !== undefined && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried === true ? (
+          `${partnerName} is my ${sex === 'f' ? (
+            'husband'
+          ) : (
+            'wife'
+          )}`
```

Comment:
Try to avoid ternary nesting. Move this expression to variable with meaningful variable naming